semgrep_output_v1.atd

```diff
@@ -586,6 +595,8 @@ type cli_match_extra <ocaml attr="deriving show"> = {
   (* EXPERIMENTAL: For now, present only for taint findings. May be extended to others
    * later on. *)
   ?dataflow_trace: cli_match_dataflow_trace option;
+
+  engine_kind: engine_kind;
```

Comment:
This part should have a question mark before! You might break consumer of this!
See the toplevel comment in this file:
```
 * This file has the _v1 suffix to explicitely represent the
 * version of this JSON format. If you need to extend this file, please
 * be careful because you may break consumers of this format (e.g., the
 * Semgrep playground or Semgrep App backend or external users of this JSON). See
 * https://atd.readthedocs.io/en/latest/atdgen-tutorial.html?highlight=json#smooth-protocol-upgrades
 * for more information on how to smoothly extend the types in this file.
```